with-camera/App.js

```diff
@@ -0,0 +1,116 @@
+import React, { useState, useEffect } from 'react';
+import { Text, View, TouchableOpacity, ImageBackground, Platform } from 'react-native';
+import { Camera } from 'expo-camera';
+
+export default function App() {
+  const [hasPermission, setHasPermission] = useState(null);
+  const [type, setType] = useState(Camera.Constants.Type.back);
+  const [lastPhotoURI, setLastPhotoURI] = useState(null);
+
+  useEffect(() => {
+    (async () => {
+      if (Platform.OS == "web") { 
+        setHasPermission(true);
+        return; 
+      }
```

Comment:
```suggestion
```
Camera permission is required on web.

Reply:
Indeed, but the web's camera permissions seem to be out of sync with (or not work well with) Camera.requestPermissionsAsync... Basically, without this branching, my browser (chrome) never asked me for cam permission and the page just stayed blank (Its what I return when there are no permissions). With this branching, I have access to the app's UI (where the Camera component get's rendered) and only then, my browser asks me for permissions independently of Camera.requestPermissionsAsync.

Ever saw something like this before ? Might be a bug with expo-camera

Reply:
Actually can't repro the behavior mentioned above, What I'm currently getting without the branching is : `Unhandled Rejection (TypeError): CameraManager.requestPermissionsAsync is not a function`

![image](https://user-images.githubusercontent.com/46449697/86061960-b4593680-ba35-11ea-8484-f937b8dd94f2.png)